jsonschema-validation.xml

```diff
@@ -740,6 +740,25 @@
                 </t>
             </section>
 
+            <section title='"readOnly"'>
+                <t>
+                    The value of this keyword MUST be a boolean.
+                </t>
+                <t>
+                    If it has a value of boolean true, this keyword indicates that the value of the
+                    instance is managed exclusively by the owning authority, and attempts by a user
+                    agent to modify the value of this property are expected to be ignored or
```

Comment:
"user agent" is a hypermedia thing.  Probably replace with "application", which is more generic?